web/experiments/feature_view.py

```diff
@@ -0,0 +1,240 @@
+import datetime
+import logging
+import os
+import re
+import sys
+from collections import defaultdict
+
+import numpy as np
+import pandas as pd
+import seaborn as sns
+from matplotlib import pyplot as plt
+from sklearn.linear_model import SGDClassifier
+from sklearn.metrics import f1_score, make_scorer
+from sklearn.model_selection import LeaveOneOut, GridSearchCV
+from tqdm import tqdm
+
+
+def process_CSLB(*embs, feature_matrix_path='../CSLB/feature_matrix.dat'):
```

Comment:
Remove the default value for feature_matrix_path argument

Reply:
And also prefix with a "_"

Reply:
Maybe better idea is change path to './feature_matrix.dat' ?